shell/lib/box.sh

```diff
@@ -44,10 +46,10 @@ download_box() {
 
   # Why: OK with assigning without checking exit code.
   # shellcheck disable=SC2155
-  local newBox=$(yq . "${BOXPATH}" |
+  local newBox=$("$YQ" . "${BOXPATH}" |
     jq --slurpfile boxconf \
```

Comment:
**thought:** Could probably remove the `jq` dependency here at some point also, right?

Reply:
I would need to figure out what the heck `--slurpfile` does first :sweat_smile: